source/conf.py

```diff
@@ -1,23 +1,34 @@
-import sys, os
+import os
+import sys
+import subprocess
 import setuptools_scm  # Just fail the process if this can't be found.
 
-# If extensions (or modules to document with autodoc) are in another directory,
-# add these directories to sys.path here. If the directory is relative to the
-# documentation root, use os.path.abspath to make it absolute, like shown here.
-# sys.path.append(os.path.abspath('.'))
+# add to the path so that sphinx can find our custom extension
+sys.path.append(os.path.abspath('../extensions/investspec'))
 
 # -- General configuration -----------------------------------------------------
+print('in conf.py')
+if not os.path.exists('../invest-sample-data'):
+    print('make get sampledata')
+    subprocess.run(['make', '-C', '..', 'get_sampledata'])
+    print('done')
+if not os.path.exists('invest-sample-data/pollination/landcover_biophysical_table_modified.csv'):
+    print('make prep sampledata')
+    subprocess.run(['make', '-C', '..', 'prep_sampledata'])
+    print('done')
```

Comment:
This was just because of me not understanding Makefiles 😅 I think I've fixed this so that `make html` will check out and prepare the sample data if needed.